### ISSUE TITLE
reprocessering på gpsftp5 sender rinex3 filer til gpsftp4, hvor de havner i saved/stale

### ISSUE THREAD

**Issue by nho-sdfe** [Kortforsyningen/GORM#30]
![repro](https://user-images.githubusercontent.com/55193081/79217890-e0f6ac80-7e4f-11ea-8e5f-b9ad235e8c31.jpg)

**Comment by kbevers** [comment]
@nho-sdfe Kan du ikke lige give lidt mere på den her. Hvad skulle der være sket med de filer?

**Comment by nho-sdfe** [comment]
Reprocessering på gpsftp5 bør ikke udløse at der sendes rinex3 filer til gpsftp4. Det er Septentrio data, hvor rådata jo er sat til upload til gpsftp4 fra /work. Ved reprocesseing bliver der dannet en del rinex3 filer i /work, som ikke skal havne på gpsftp4

**Comment by kbevers** [comment]
Tak, Henrik. Jo mere udførlige fejlrapporter du kan lave jo nemmere bliver det for Tanya og/eller Søren at finde en løsning.

**Comment by TanyaPS** [comment]
Er det fordi den burde uploade til gpsftp4 fra /ftp? Ellers skal den vel kun uploade filerne når de først kommer til /work og ikke efter de e rblevet lavet om til rinex3?

**Comment by kbevers** [comment]
@nho-sdfe vil du være venlig at svare Tanya

**Comment by nho-sdfe** [comment]
Tanya, jeg forstår ikke helt dit spørsmål, men reprocessering på gpsftp5 bør ikke udløse at der bliver sendt filer til gpsftp4.

**Comment by TanyaPS** [comment]
jeg kan ikke forstå hvordan det kun er et problem når filerne reprocesseres

**Comment by nho-sdfe** [comment]
Nej, det kan jeg heller ikke. Men når sbf filerne kommer ind på gpsftp5 første gang bliver kun sbf filerne sendt til gpsftp4 og det er som det skal være. Hvad forskellen så er når man reprocessere, kan jeg ikke gennemskue.

**Comment by casperjepsen** [comment]
Skal løses ved at kode fra gpsftp4 skal integreres på gpsftp5 - så dette bør løses der...
